src/main/java/org/veupathdb/service/eda/ds/plugin/correlation/AbstractCorrelationBipartiteNetwork.java

```diff
@@ -91,7 +95,7 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
       LinkData link = new LinkDataImpl();
       link.setSource(sourceNode);
       link.setTarget(targetNode);
-      link.setStrokeWidth(correlationRow.getCorrelationCoef());
+      link.setStrokeWidth(String.valueOf(Math.abs(Float.parseFloat(correlationRow.getCorrelationCoef()))));
```

Comment:
I assume the client code scales this to something larger than [0,1]?

Naming it `strokeWidth` suggests that this is used as-is on the client side.  Why not call it `edgeWeight` or something like that? (not urgent, of course)

Reply:
good idea bob! Actually by a lucky chance i've added you as a reviewer on the PR that does the scaling. I like your point about how since it gets scaled, it's not really strokeWidth.